src/components/higlass/HiGlassSubscriber.js

```diff
@@ -0,0 +1,63 @@
+import React, { useMemo, useEffect, useCallback } from 'react';
+import loadable from '@loadable/component';
+import TitleInfo from '../TitleInfo';
+
+const HiGlassComponent = loadable(() => import('higlass').then(d => new Promise(resolve => resolve(d.HiGlassComponent))));
+
+/**
+ * A wrapper around HiGlass (http://higlass.io/).
+ * The HiGlassComponent react component is loaded lazily.
+ * @prop {object} hgViewConfig A HiGlass viewconfig object to pass
+ * to the HiGlassComponent viewConfig prop.
+ * @prop {object} hgOptions An optional HiGlass object to pass
+ * to the HiGlassComponent hgOptions prop.
+ * @prop {function} removeGridComponent A grid component removal handler
+ * to pass to the TitleInfo component.
+ * @prop {function} onReady A callback function to signal that the component is ready.
+ */
+export default function HiGlassSubscriber(props) {
+  const {
+    hgViewConfig,
+    hgOptions = {
+      bounded: true,
+      pixelPreciseMarginPadding: true,
+      containerPaddingX: 0,
+      containerPaddingY: 0,
+      sizeMode: 'default',
+    },
+    removeGridComponent,
+    onReady,
+  } = props;
+
+  const onReadyCallback = useCallback(onReady, []);
+
+  useEffect(() => {
+    onReadyCallback();
+  }, [onReadyCallback]);
```

Comment:
I think usually we have a `clearPleaseWait` for each component, is there not a `console.warn` for higlass? I'm looking for something like:

```javascript
PubSub.publish(CLEAR_PLEASE_WAIT, 'higlass');
```

Reply:
I think the console warnings are only for things listed in the vitessce view config `layers` array.

Reply:
Ah yes! Thanks.

Reply:
In the future maybe there will be something in layers that we want higlass to know about, but I think we can wait until that is more clear to implement the clear please wait.

Reply:
👍 : Yeah, for now, the whole hg-viewconf is just embedded, and trying to figure out how to wait for HG's internal requests is way out of scope.